src/meta/partition_guardian.cpp

```diff
@@ -72,8 +72,26 @@ pc_status partition_guardian::on_missing_secondary(meta_view &view, const dsn::g
 
 pc_status partition_guardian::on_redundant_secondary(meta_view &view, const dsn::gpid &gpid)
 {
-    // TBD(zlw)
-    return pc_status::invalid;
+    const node_mapper &nodes = *(view.nodes);
+    const partition_configuration &pc = *get_config(*(view.apps), gpid);
+    int target = 0;
+    int load = nodes.find(pc.secondaries.front())->second.partition_count();
+    for (int i = 0; i != pc.secondaries.size(); ++i) {
+        int l = nodes.find(pc.secondaries[i])->second.partition_count();
```

Comment:
what is the `l` means

Reply:
I have told you that I don't want to modify any code in this pull request